.travis.yml

```diff
@@ -17,6 +17,8 @@ language: java
 
 sudo: false
 
+dist: precise
```

Comment:
The build was [failing on travis](https://travis-ci.org/sanjaydasgupta/zeppelin/builds/256845905) without it, and there was a comment in the travis mail suggesting this course of action.